### PR TITLE
Reduce analytics sampling from 100 to 70 per cent

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[Google Analytics client-side sampling (2)](https://trello.com/c/uG8F1Arh/6347-google-analytics-client-side-sampling-2)

File: orcid-web/src/main/resources/freemarker/layout/google_analytics.ftl [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-	ga('create', '${googleAnalyticsTrackingId}', 'auto', {'legacyCookieDomain': 'orcid.org'});  // Replace with your property ID.
+	ga('create', 'UA-0000000-00', 'auto', {'legacyCookieDomain': 'orcid.org', 'sampleRate': '70' });
```
Comment (amontenegro) [inline orcid-web/src/main/resources/freemarker/layout/google_analytics.ftl]: @leomendoza123 is it ok that the id is hardcoded here?

Comment (leomendoza123) [reply orcid-web/src/main/resources/freemarker/layout/google_analytics.ftl]: Great catch, I am updating this

**Comment by leomendoza123** [comment]
please NOT merge this PR until https://github.com/ORCID/orcid-angular/pull/98 is merged

**Pull request merged**